mvlearn/embed/base.py

```diff
@@ -68,13 +67,248 @@ def fit_transform(self, Xs, y=None):
         Xs : list of array-likes or numpy.ndarray
              - Xs length: n_views
              - Xs[i] shape: (n_samples, n_features_i)
+
         y : array, shape (n_samples,), optional
             Targets to be used if fitting the algorithm is supervised.
 
         Returns
         -------
-        X_transformed : list of array-likes
+        X_transformed : list of numpy.ndarray
             - X_transformed length: n_views
             - X_transformed[i] shape: (n_samples, n_components_i)
         """
         return self.fit(Xs=Xs, y=y).transform(Xs=Xs)
+
+
+class BaseCCA(BaseEstimator, TransformerMixin):
+    """
+    A base class for multiview CCA methods.
+    """
+
+    def fit(self, Xs, y=None):
+        r"""
+        Learns decompositions of the views.
+
+        Parameters
+        ----------
+        Xs : list of array-likes or numpy.ndarray
+             - Xs length: n_views
+             - Xs[i] shape: (n_samples, n_features_i)
+            The data to fit to.
```

Comment:
This one also doesn't render well for some reason. The second bullet point has an extra indent in rendering. Maybe for all things like this we should put the line like "The data to fit to" above the bullet points